application/models/thread.go

```diff
@@ -197,3 +197,20 @@ func (t *Thread) Load(fields ...string) error {
 
 	return nil
 }
+
+func (t *Thread) UnreadMessageCount(lastViewedAt time.Time) (int, error) {
+	count := 0
+
+	msgs, err := t.GetMessages([]string{"created_at"})
+	if err != nil {
+		return count, err
+	}
+
+	for _, m := range msgs {
+		if m.CreatedAt.After(lastViewedAt) {
```

Comment:
Wondering if this should be `UpdatedAt`. Doesn't matter today, but I'm curious about what happens if we allow editing of old messages.

Reply:
I checked with Billy.  He doesn't think we will end up allowing users to edit messages.